### DISCUSSION TITLE
Thumbnail view does not respect Exif orientation metadata

### DISCUSSION THREAD

**Discussion by psanda** [doublecmd/doublecmd#1171] [Suggestions]
Thumbnail view of pictures does not respect Exif orientation metadata which makes it ugly for quick viewing photos from cameras.

**Comment by Skif-off** [comment]
@alexx2000 If you add support for the orientation flag in this case too, can you do it optional?

**Comment by WhiteChairFromIkea** [comment]
> @alexx2000 If you add support for the orientation flag in this case too, can you do it optional?

This is user depandable, I do not like autorotated thumbnails, making this optional would be all winner.

**Comment by alexx2000** [comment]
Yes, it can be optional.

**Comment by cbhushan** [comment]
Parts of this pigallery2 discussion are potentially relevant to this issue: https://github.com/bpatrik/pigallery2/issues/212

**Comment by Skif-off** [comment]
"incorrect aspect ratio" and "ignoring tag" are very different things :) Bug-report != feature request...

**Comment by CristianKerr** [comment]
Hi, I would also love to have an option to set the thumbnail view to respect Orientation exif tag. 😊
